scripts/compare_fsoi.py

```diff
@@ -59,14 +59,15 @@ def sort_centers(DF,pref):
         df.append(tmp)
     return df
 
-def main():
+def compare_fsoi_main():
 
     parser = ArgumentParser(description = 'Create and Plot Comparison Observation Impact Statistics',formatter_class=ArgumentDefaultsHelpFormatter)
-    parser.add_argument('--rootdir',help='root path to directory',type=str,default='/scratch3/NCEPDEV/stmp2/Rahul.Mahajan/test/Thomas.Auligne/FSOI',required=False)
+    parser.add_argument('--rootdir',help='root path to directory',type=str,required=True)
     parser.add_argument('--platform',help='platforms to plot',type=str,default='full',choices=['full','conv','rad'],required=False)
-    parser.add_argument('--cycle',help='cycle to process',nargs='+',type=int,default=[0],choices=[0,6,12,18],required=False)
-    parser.add_argument('--norm',help='metric norm',type=str,default='dry',choices=['dry','moist'],required=False)
+    parser.add_argument('--cycle',help='cycle to process',nargs='+',type=int,choices=[0,6,12,18],required=True)
+    parser.add_argument('--norm',help='metric norm',type=str,choices=['dry','moist'],required=True)
     parser.add_argument('--savefigure',help='save figures',action='store_true',required=False)
+    parser.add_argument('--centers', help='comma-separated list of centers', type=str, required=True)
```

Comment:
Instead if you add nargs='+', you won't need to do the split(',') and strip on line 79.
The input argument is then simply --centers GMAO EMC NRL.
Also, provide a choices=['GMAO', 'NRL', ...] to ensure that correct centers are provided.

Reply:
Thanks for the suggestion here.  I have made this change.